tomcat/src/main/docker/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM ${docker.openjdk.image}
+
+# Create tomcat group and user
+RUN groupadd -r tomcat \
+ && useradd -r -g tomcat tomcat
+
+# Environment variable for tomcat
+ENV CATALINA_HOME ${tomcat.home}
+RUN mkdir -p $CATALINA_HOME
+
+# Create Tomcat home
+ADD tomcat-home $CATALINA_HOME
+RUN chmod +x ${CATALINA_HOME}/bin/startup.sh ${CATALINA_HOME}/bin/catalina.sh
```

Comment:
Can you combine as many `RUN` commands into one as possible with `&&` to avoid creating too many layers in the docker image?